### DISCUSSION TITLE
Change server location

### DISCUSSION THREAD

**Discussion by pouyaw** [angristan/openvpn-install#847] [Help]
**⚠️ Unless you are sure you find a bug with the script, please open a [discussion](https://github.com/angristan/openvpn-install/discussions) instead of an issue!**

**Checklist**

- [x] I read the [README](https://github.com/angristan/openvpn-install/blob/master/README.md)
- [x] I read the [FAQ](https://github.com/angristan/openvpn-install/blob/master/FAQ.md)
- [x] I searched the [issues](https://github.com/angristan/openvpn-install/issues?q=is%3Aissue+)
- [x] I searched the [discussion](https://github.com/angristan/openvpn-install/discussions)
- [ ] My issue is about the script, and not OpenVPN itself

Pease include as much details as possible in your issue:

- Description of the issue
- How to reproduce the issue
- What did you expected should happen
- Logs
- Server/Client versions (OS, OpenVPN, etc)
- Any context or information that could help

---

Hi, I transfer my vps to new server with new ip. Now how can i change my ip? I changed from server.conf it's not effective

**Comment by angristan** [comment]
Did you update the IP in your client configuration files? What error are you encountering? Please be specific when asking for help, other people's time is precious :)
